.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release to NPM and PyPI
+# Builds and releases the application to NPM and PyPI 
+# when a new release is published or the workflow is manually triggered.
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+env:
+  # SNYK_TOKEN: xxxxa727
+  TWINE_NON_INTERACTIVE: true
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  TWINE_PASSWORD: ${{ secrets.TWINE_PASSWORD }}
+  TWINE_REPOSITORY_URL: ${{ secrets.TWINE_REPOSITORY_URL }}
+  TWINE_USERNAME: ${{ secrets.TWINE_USERNAME }}
+jobs:
+  release_to_npm:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4  
+
+    - uses: actions/setup-node@v4
+      with:
+        node-version: 18.20.0
```

Comment:
[Nit] This is good, but if there is any way to have some config yaml file where we read these versions for each workflow (incase the versions are different), it would be easier for maintenance. For example -
```yaml
workflow_config:
  release:
   python: 3.9
   node: 18.20.0
  tests:
    backend_unit_tests:
      python: 3.8
      node: 18.20.0
    frontend_unit_tests:
     ...
```

Reply:
For now GitHub Actions does not support external configuration files for defining versions or other variables. For workflow they do support setting ENV variable at workflow top level and they have beta feature for [multiple workflow](https://docs.github.com/en/actions/learn-github-actions/variables#defining-configuration-variables-for-multiple-workflows) also.
For now, I am setting ENV variable (node version, python version, timeout & interval) at top level for each workflow, later we can have a separate ticket for this to set for all the workflows.